### PR TITLE
Internationalization and Localization - Part 1

### PR DESCRIPTION
**Pull request by yulieth9109**
This pull is to Include the internationalization package and the 
Generated arb files with the keys of the text translated.

### CODE REVIEW

File: l10n.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+arb-dir: lib/l10n
+template-arb-file: app_de.arb
+output-localization-file: app_localizations.dart
```
Comment (7h30n3) [inline l10n.yaml:3]: Add an empty line at the end of the file.

**Comment by 7h30n3** [comment]
All tags should be English, not a mix between English and German like `about_quellcode`.

File: lib/l10n/app_de.arb [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+        "type": "num",
+        "format": "compact"
+      }
+    }
+  },
+  "number_input_value": "Wert",
+  "@number_input_value": {},
+  "onboarding_button1": "So funktioniert''s",
+  "@onboarding_button1": {},
+  "onboarding_button2": "Mach'' ich",
```
Comment (7h30n3) [inline lib/l10n/app_de.arb:125]: Only use one apostrophe. Check the whole file for it.

File: lib/l10n/app_en.arb [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  "@helpReportError": {},
+  "helpTitle": "Help",
+  "@helpTitle": {},
+  "homeSidebarLogOutOSM": "Log out of OSM?",
+  "@homeSidebarLogOutOSM": {},
+  "homeSidebarAbort": "Abort",
+  "@homeSidebarAbort": {},
+  "homeSidebarLogOut": "Log out",
+  "@homeSidebarLogOut": {},
+  "homeSidebarLogin": "Login",
```
Comment (Robbendebiene) [inline lib/l10n/app_en.arb:49]: "Login" > "Log in"

File: lib/l10n/app_en.arb [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+  "@homeSidebarLogin": {},
+  "homeSidebarSettings": "Settings",
+  "@homeSidebarSettings": {},
+  "homeSidebarAbout": "About",
+  "@homeSidebarAbout": {},
+  "homeSidebarHelp": "Help",
+  "@homeSidebarHelp": {},
+  "homeSidebarUserMessajeLogout": "If you log out, you can no longer upload changes to OpenStreetMap.",
+  "@homeSidebarUserMessajeLogout": {},
+  "homeSidebarUserMessajeLogin": "Login with your OpenStreetMap account to upload your changes.",
```
Comment (Robbendebiene) [inline lib/l10n/app_en.arb:59]: "Login" > "Log in"

File: lib/l10n/app_en.arb [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+  "@numberInputMaxLimit": {},
+  "numberInputMinLimit": "{nameString} must be greater than {min}.",
+  "@numberInputMinLimit": {},
+  "numberInputValue": "Value",
+  "@numberInputValue": {},
+  "onboardingScrollButtonSection1": "Here's how it works",
+  "@onboardingScrollButtonSection1": {},
+  "onboardingScrollButtonSection2": "I'll do it",
+  "@onboardingScrollButtonSection2": {},
+  "onboardingScrollButtonSection3": "OK, got it",
```
Comment (Robbendebiene) [inline lib/l10n/app_en.arb:75]: OK > Okay

File: lib/l10n/app_en.arb [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+  "@onboardingScrollButtonSection3": {},
+  "onboardingScrollButtonSection4": "Here we go",
+  "@onboardingScrollButtonSection4": {},
+  "onboardingDescriptionSection1": "We are pleased that you are here and that you want to do your part to improve local transport.",
```
Comment (Robbendebiene) [inline lib/l10n/app_en.arb:79]: "We're glad you're here and want to do your part to improve public transport."

File: lib/l10n/app_en.arb [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+  "@onboardingDescriptionSection1": {},
+  "onboardingDescriptionSection2": "Go to a nearby stop to see its current status.",
+  "@onboardingDescriptionSection2": {},
```
Comment (Robbendebiene) [inline lib/l10n/app_en.arb:82]: "Go to a nearby stop to survey its current state."

File: lib/l10n/app_en.arb [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+  "onboardingDescriptionSection3": "Select a map marker in the app to capture and answer the questions that appear.",
```
Comment (Robbendebiene) [inline lib/l10n/app_en.arb:83]: "In order to collect data select a marker in the app and answer the displayed questions."

File: lib/l10n/app_en.arb [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+  "@onboardingDescriptionSection3": {},
+  "onboardingDescriptionSection4": "Upload your answers to OpenStreetMap and make them available to the whole world.",
```
Comment (Robbendebiene) [inline lib/l10n/app_en.arb:85]: "Upload your answers to OpenStreetMap to share them with the whole world."

File: lib/l10n/app_en.arb (hunk 8 above)
Comment (Robbendebiene) [inline lib/l10n/app_en.arb:83]: "In order to collect data select a marker in the app and answer the displayed questions."

File: lib/l10n/app_en.arb [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+  "@questionDialogContinue": {},
+  "questionDialogButtonBack": "Back",
+  "@questionDialogButtonBack": {},
+  "questionDialogButtonSkip": "Skip",
+  "@questionDialogButtonSkip": {},
+  "questionDialogUserMessageSuccess": "Changes successfully applied.",
+  "@questionDialogUserMessageSuccess": {},
+  "questionDialogErrorMessageConnection": "Error: No connection to the OSM server.",
+  "@questionDialogErrorMessageConnection": {},
+  "questionDialogErrorMessageTransmission": "Unknown error during transmission",
```
Comment (Robbendebiene) [inline lib/l10n/app_en.arb:109]: "Unknown error during transmission."

File: lib/l10n/app_en.arb [hunk 11]
```diff
@@ -0,0 +1,139 @@
...
+  "@settingsThemeOptionDark": {},
+  "settingsThemeOptionUnknown": "Unknown",
+  "@settingsThemeOptionUnknown": {},
+  "settingsMenuTitle": "Settings",
+  "@settingsMenuTitle": {},
+  "settingsMenuOptionColor": "Color Scheme of the App",
+  "@settingsMenuOptionColor": {},
+  "settingsTitleTheme": "Select theme",
+  "@settingsTitleTheme": {},
+  "settingsMenuOptionAdvanceQuestions": "Display Advanced Questions",
```
Comment (Robbendebiene) [inline lib/l10n/app_en.arb:133]: "Display Professional Questions"

File: pubspec.lock
Comment (a-andre) [inline pubspec.lock:1]: IMO dependency updates should be done in another pull request.

File: lib/l10n/app_de.arb (hunk 2 above)
Comment (yulieth9109) [reply lib/l10n/app_de.arb]: Done

**Review by Robbendebiene — APPROVED** [review]
LGTM
